src/scarab.js

```diff
@@ -110,11 +110,36 @@ var MapzenScarab = (function () {
     el.appendChild(mapzenEl)
     el.appendChild(twitterEl)
     el.appendChild(facebookEl)
-    el.appendChild(githubEl)
+    
+    // Creating github icon button if needed
+    if (opts.repo) {
+      var githubEl = _createButtonEl('github', githubLink, 'View source on GitHub', _onClickGitHub)
```

Comment:
Use `opts.repo` instead of `githubLink` because they are the same.
Also, because `githubEl` is being created conditionally here, we should remove it from being created in line 107.